src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,45 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        Driver driverOne = new Driver("Giwi", "111");
+        Driver driverTwo = new Driver("Gogi", "222");
+        Driver driverThree = new Driver("Koba", "555");
+        Driver driverFour = new Driver("Ashan", "666");
+        driverService.create(driverOne);
```

Comment:
You can add them to list and create them in for loop or in stream or forEach loop. The same for other objects.

Reply:
I don't understand why we need to create a list and loop? What are the improvements from this?